pkg/runner/command.go

```diff
@@ -153,8 +153,35 @@ func (cr *CommandRunner) handleInternalCmd() (int, string) {
 			return 1, fmt.Sprintf("openftp: Not enough information. %s", err.Error())
 		}
 
-		ftpClient := NewFtpClient(cr.data)
-		go ftpClient.RunFtpBackground()
+		sysProcAttr, err := demote(data.Username, data.Groupname)
```

Comment:
Too much logic in the "openftp" case block. Recommend refactoring into separate functions for readability. Also, avoid reassigning variables like `err` multiple times in the same block.